.github/workflows/binaries.yml

```diff
@@ -2,10 +2,10 @@ on:
   push:
     paths-ignore: ['media/**', 'docs/**', '**/*.md']
     branches:
-    - master
-    pull_request:
-      paths-ignore: ['media/**', 'docs/**', '**/*.md']
-    workflow_dispatch:
+      - master
+  pull_request:
+    paths-ignore: ['media/**', 'docs/**', '**/*.md']
```

Comment:
Do we actually want to run this workflow for pull requests? Seems like you'd only want it for pushes to `master`?

Reply:
I could also just add an `if` into the release publishing so it doesn't try and publish for a PR